server/network/aoprotocol.py

```diff
@@ -950,13 +950,20 @@ def net_cmd_mc(self, args):
         if not self.client.is_checked:
             return
 
-        if args[0].split('\n')[0] == "{ Areas }":
+        if args[0].split(':')[0] == "[HUB":
             # self.client.send_ooc('Switching to the list of Hubs...')
             self.client.viewing_hub_list = True
             preflist = self.client.server.supported_features.copy()
             preflist.remove('arup')
             self.client.send_command('FL', *preflist)
-            self.client.send_command('FA', *['{ Hubs }\n Double-Click me to see Areas\n  _______', *[f'[{hub.id}] {hub.name}' for hub in self.client.server.hub_manager.hubs]])
+            for thub in self.client.server.hub_manager.hubs:
```

Comment:
thub? What's t stand for?
also, c in KFO-Server code usually is a short-hand for "client" - though that's also not ideal. One letter variables are hard to understand.
maybe just change "c" to "count"?